.github/workflows/build.yml

```diff
@@ -37,12 +39,12 @@ jobs:
     strategy:
       matrix:
         config:
-          - { cxx: clang++, image: '2019', abi: '6', build: 'Release',     test: 'ON'  }
-          - { cxx: clang++, image: '2020', abi: '7', build: 'Release',     test: 'ON'  }
-          - { cxx: clang++, image: '2021', abi: '8', build: 'Release',     test: 'ON'  }
-          - { cxx: clang++, image: '2021', abi: '8', build: 'DebugNoInfo', test: 'OFF' } # tests take too long..
```

Comment:
The "DebugNoInfo" mode was because the job would occasionally run out of disk space when building the Houdini plugins in debug mode with all symbols:

https://github.com/AcademySoftwareFoundation/openvdb/blame/master/tsc/meetings/2020-04-02.md#L56

The limit is still 14GB of disk space and I believe we are over that if we enable symbols again, so to change this would just rely on not having the limit strictly enforced?

Reply:
This workflow doesn't build the houdini plugins - and interestingly, the Houdini workflow in houdini.yml uses 'Debug'! This might have been a mistake when moving this around, but they're not failing. I think it's okay to change this one toDebug but, as the Houdini stuff takes so long, it might be ever so slightly faster to put that back to `DebugNoInfo`